src/nodes.jl

```diff
@@ -335,13 +332,14 @@ function run!(op::Op)
         end
     end
 
-    kwargs = map(op.kwargs) do kwarg
+    kwargs = [
```

Comment:
What did you learn about the map not working?

Reply:
I learned that I have the memory of a goldfish. I figured out that the problem was that `op.kwargs` was a `Base.Iterators.Pairs{Symbol,Int64,Tuple{Symbol},NamedTuple{(:limit,),Tuple{Int64}}}` and I could just collect it, and then I forgot to actually put the fix in the code.